examples/ol3.html

```diff
@@ -81,7 +81,7 @@ <h1 class="sidebar-header">Settings<span class="sidebar-close"><i class="fa fa-c
 
     <a href="https://github.com/Turbo87/sidebar-v2/"><img style="position: fixed; top: 0; right: 0; border: 0;" src="https://s3.amazonaws.com/github/ribbons/forkme_right_darkblue_121621.png" alt="Fork me on GitHub"></a>
 
-    <script src="//openlayers.org/en/v4.6.5/build/ol.js" type="text/javascript"></script>
+    <script src="//cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@master/en/v6.1.1/build/ol.js"></script>
```

Comment:
I would prefer to use the paths that are listed on https://openlayers.org/download/ here

```suggestion
    <script src="https://cdn.rawgit.com/openlayers/openlayers.github.io/master/en/v6.1.1/build/ol.js"></script>
```

Reply:
I changed the path to avoid rawgit. According to https://rawgit.com main page "RawGit has reached the end of its useful life" and jsdelivr.net is one of the alternative you can choose (also detailed in the same page)
So, I need to be sure before changing commit to use rawgit instead.
What's your opinion now? Thanks

Reply:
oh, interesting. didn't know about that...

I guess we can keep it like this then. are the openlayers people aware of this? might make sense to open a PR on their project that changes the URLs there too.